### DISCUSSION TITLE
Search for open Pull-requests of a team in an organisation

### DISCUSSION THREAD

**Discussion by tbille** [community/community#34318] [Pull Requests]
:wave: 

in the search page of GitHub is there a way to filter issues/pull-requests/other by a team part of an organisation only?

For example: in the org "solar-system" there is the team "earth". The team earth manages multiple repositories. As a member of earth I want to see all current open pull requests of my team. Is there a filter like this existing?

```
org:solar-system team:earth is:pr is:open
```
or
```
org:solar-system/earth is:pr is:open
```

**Comment by manasag** [comment]
+1 to this feature, especially in the Pull Request tab search. As an alternate even the support of multiple authors in the search can solve this (albeit more work). Currently I am using a multi author search on the main Github search box, but the result view is not tabulated or optimized to view PRs.
